Sources/Rendering/Core/ImageMapper/index.js

```diff
@@ -113,6 +113,30 @@ function vtkImageMapper(publicAPI, model) {
     return image.extentToBounds(ex);
   };
 
+  publicAPI.getBoundsForSlice = (slice, thickness = 0) => {
```

Comment:
@martinken @jourdain I need this on the ImageSlice also to my understanding, to apply the necessary transform

Reply:
ok

Reply:
I don't quite understand how that is working here... https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageSlice/index.js#L56-L102

Reply:
done, ready